### PR TITLE
Config utilities ts

### PR DESCRIPTION
**Pull request by RDiMag**

### CODE REVIEW

File: src/app/dao/plan.dao.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 const { ObjectId } = require('mongodb');
-const { defaultProfilePicture } = require('../../config/defaultVars.config');
+const { defaultProfilePicture } = require('../../config/defaultVars.config').default;
```
Comment (theodorusandi) [inline src/app/dao/plan.dao.js]:
```
import cube from './my-module.js';

```

File: src/utilities/password.util.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import crypto = require('crypto');
```
Comment (theodorusandi) [inline src/utilities/password.util.ts]: `import cube from './my-module.js';`

use either CJS/ESM

File: src/utilities/ses-transport.util.ts [hunk 3]
```diff
@@ -1,5 +1,5 @@
 const nodemailer = require('nodemailer');
-const aws = require('@aws-sdk/client-ses');
+import aws = require('@aws-sdk/client-ses');
```
Comment (theodorusandi) [inline src/utilities/ses-transport.util.ts]: same as above

File: src/utilities/password.util.ts (hunk 2 above)
Comment (RDiMag) [reply src/utilities/password.util.ts]: When I use this syntax, I'm getting an error:
`import crypto from 'crypto';
`

The error says "Module '"crypto"' has no default export."

File: src/utilities/ses-transport.util.ts (hunk 3 above)
Comment (RDiMag) [reply src/utilities/ses-transport.util.ts]: I am also getting the error "Module... has no default export." with this syntax:
`import aws from '@aws-sdk/client-ses';`

I also tried this, with the same error:
`import aws from '../../node_modules@aws-sdk/client-ses';`

Comment (RDiMag) [reply src/utilities/ses-transport.util.ts]: Solution was to do `import * as aws from...`

**Review by theodorusandi — APPROVED** [review]

**Review by jpatrici — APPROVED** [review]

**Pull request merged**